src/core/server/logging/logger.ts

```diff
@@ -139,7 +139,7 @@ export class BaseLogger implements Logger {
     const supportedLevel = this.level.supports(record.level);
 
     for (const appender of this.appenders) {
-      if (supportedLevel || appender.receiveAllLevels) {
+      if (supportedLevel) {
         appender.append(record);
       }
     }
```

Comment:
Can probably change this back to how it was before #55752 and return early if `!this.level.supports(record.level)` rather than do this check on each loop iteration